IFC4x3/Properties/d/DipDirection_17WEfvK3L9Ehv8Rdn5AK_c/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DipDirection_17WEfvK3L9Ehv8Rdn5AK_c" Name="DipDirection" UniqueId="4780ea79-5035-493a-be48-6e7c45294fa6" PrimaryDataType="IfcPlaneAngleMeasure" />
```

Comment:
Seems like ambiguous data since a fault should have a geometric representation. FYI this is manual input since it is a property. Please consider removing this property.

Reply:
We recommedn to keep it because it adds value, giving a "main direction" for a complex geometry, commonly used that way.

Reply:
Kept and reused from Book A.